test/generator/template_values_test.rb

```diff
@@ -71,30 +71,10 @@ def paths
       include TemplateValuesFactory
     end
 
-    class ClassBasedTestTemplateValuesFactory
+    class ClassBasedTestTemplateValuesFactory < TestTemplateValuesFactory
```

Comment:
You like tests to be way more DRY than I do.
I'm not sure this is a good idea.
(But it's not wrong.)